bundle/Helper/ImportUrlsHelper.php

```diff
@@ -0,0 +1,250 @@
+                'totalUrls' => $totalUrls,
+                'return' => $return,
+                'fileLog' => $this->webDirectory."redirectUrls/logs/redirect_import_urls-".date('d-m-Y').".csv",
+            ];
+
+        }
+
+        return $params;
+    }
+
+    /**
+     * @param $destination
+     * @return \eZ\Publish\API\Repository\Values\Content\URLWildcardTranslationResult|null
+     */
+    public function checkUrlDestinationExist($destination)
+    {
+        $urlExists = null;
+
+        try {
+            $urlExists = $this->urlWildCardService->translate($destination);
+
+        } catch (\Exception $e) {
+            $this->logger->log(LogLevel::ERROR, $e->getMessage());
+        }
+
+        return $urlExists;
+    }
+
+    /**
+     * @param $source
+     * @param $destination
+     * @return array
+     */
+    public function saveUrls($source, $destination)
+    {
+        $return = $errors = [];
+        try {
+            $result = $this->urlWildCardService->create($source, $destination, 'Redirection');
+            if ($result) {
+                $return = [
+                    'source' => $source,
+                    'destination' => $destination,
+                    'msg' => $this->translator->trans('nova.import.list.table.info', [], 'redirect'),
+                    'imported' => 'OK',
+                ];
+
+                $msg = $return['msg'];
+                $status = $return['imported'];
+                $this->fs->appendToFile(
+                    $this->webDirectory."redirectUrls/logs/redirect_import_urls-".date('d-m-Y').".csv",
+                    "$source;$destination;$msg;$status\n"
+                );
+            }
+        } catch (\Exception $e) {
+            $return = [
+                'source' => $source,
+                'destination' => $destination,
+                'msg' => $e->getMessage(),
+                'imported' => 'KO',
+            ];
+            $msg = $return['msg'];
+            $status = $return['imported'];
+            $this->fs->appendToFile(
+                $this->webDirectory."redirectUrls/logs/redirect_import_urls-".date('d-m-Y').".csv",
+                "$source;$destination;$msg;$status\n"
+            );
+            $this->logger->log(LogLevel::ERROR, $e->getMessage());
+        }
+
+        return $return;
+    }
+
+    /**
+     * @param $fileToImport
+     * @param $fileLog
+     */
+    public function saveFileHistory($fileToImport, $fileLog)
+    {
+        try {
+            $redirectImportHistory = new RedirectImportHistory();
+            $redirectImportHistory->setNameFile($fileToImport->getFilename());
+            $redirectImportHistory->setDate(new \DateTime());
+            $redirectImportHistory->setPath($fileLog);
+            $this->entityManager->persist($redirectImportHistory);
+            $this->entityManager->flush();
+        } catch (\Exception $e) {
+            $this->logger->log(LogLevel::ERROR, $e->getMessage());
+        }
+    }
+
+    /**
```

Comment:
those functions do not bring any value I would remove them and use the property directly

Reply:
I did not understand, can you explain to me what you want me to change